Arduino_Sub_DAQ/SiMlx.ino

```diff
@@ -0,0 +1,75 @@
+/*************************************************** 
+  This is a library example for the MLX90614 Temp Sensor
+
+  Designed specifically to work with the MLX90614 sensors in the
+  adafruit shop
+  ----> https://www.adafruit.com/products/1748
+  ----> https://www.adafruit.com/products/1749
+
+  These sensors use I2C to communicate, 2 pins are required to  
+  interface
+  Adafruit invests time and resources providing this open source code, 
+  please support Adafruit and open-source hardware by purchasing 
+  products from Adafruit!
+
+  Written by Limor Fried/Ladyada for Adafruit Industries.  
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <Wire.h>
+#include <Adafruit_MLX90614.h>
+#define IR1 0x5C
+#define IR2 0x5A
+#define IR3 0x5B
+#define IR4 0x5D
+Adafruit_MLX90614 mlx;
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  Serial.println("Adafruit MLX90614 test");  
+
+  if(!mlx.begin()) {
+    
+    Serial.println("Error connecting to MLX sensors. Check wiring.");
+    while (1);
+      } 
+ 
+  //Serial.print("Emissivity = "); Serial.println(mlx.readEmissivity());
+  Serial.println("================================================");
+  
+}
+
+
+void loop() {
+  
+  mlx.AddrSet(IR1); 
```

Comment:
Can this be moved to `setup()`?

Reply:
if you mean moving the void loop function into the setup function, error becomes: 
Compilation error: a function-definition is not allowed here before '{' token

if you remove the void loop function and move the code block into the setup function, a .cpp file used in one of the libraries makes a reference to a loop: C:\Users\denni\AppData\Local\Arduino15\packages\arduino\hardware\avr\1.8.6\cores\arduino/main.cpp:46: undefined reference to `loop'
collect2.exe: error: ld returned 1 exit status

Reply:
My apologies, I meant moving `.AddrSet()`

Reply:
Compiling error: \AppData\Local\Arduino15\packages\arduino\hardware\avr\1.8.6\cores\arduino/main.cpp:46: undefined reference to `loop'
Presumably one of the other libraries that the code relies on makes a reference to this loop, is my understanding. I could take a look at the .cpp file in the future, though that will have to be on another day.